packages/react-components/react-timepicker-compat-preview/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -95,11 +95,7 @@ export type TimeFormatOptions = {
 /**
  * TimePicker Props
  */
-export type TimePickerProps = Omit<
-  ComponentProps<Partial<ComboboxSlots>, 'input'>,
-  | 'children' // TODO add children prop to allow custom children through render function
```

Comment:
remove this TODO because we decide to not implement this unless someone request it